gsc/_utils.scm

```diff
@@ -1006,5 +1001,20 @@
           (not (member x b)))
         a))
 
+(define (lset-union a b)
```

Comment:
I don't see that you use this procedure anywhere.  Perhaps I'm missing something.